hazelcast/src/main/java/com/hazelcast/instance/HazelcastInstanceProxy.java

```diff
@@ -68,7 +68,7 @@
  * </li>
  * </ol>
  */
-public final class HazelcastInstanceProxy implements HazelcastInstance, SerializationServiceSupport {
+public class HazelcastInstanceProxy implements HazelcastInstance, SerializationServiceSupport {
     protected volatile HazelcastInstanceImpl original;
```

Comment:
For the back compatibility I need to return something which would extend HazelcastInstanceProxy but with JET functionality